docs/src/pages/[platform]/components/phonenumberfield/examples/ThemingExample.tsx

```diff
@@ -1,12 +1,16 @@
-import { ThemeProvider, PhoneNumberField, Theme } from '@aws-amplify/ui-react';
+import { ThemeProvider, PhoneNumberField } from '@aws-amplify/ui-react';
 
-const theme: Theme = {
+const theme = {
   name: 'phonenumber-theme',
   tokens: {
     components: {
-      fieldcontrol: {
+      phonenumberfield: {
         color: { value: '{colors.blue.60}' },
         fontSize: { value: '{fontSizes.large}' },
+        borderColor: { value: '{colors.neutral.80}' },
+        _focus: {
+          borderColor: { value: '{colors.neutral.80}' },
```

Comment:
Can we make the borderColor in the _focus state different than the default borderColor `{colors.neutral.80}`?

Reply:
updated